### PR TITLE
feat: added python projects

### PR DESCRIPTION
**Pull request by SherbetLemon47**
For #2

### CODE REVIEW

File: docs/python.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+- [Arrow](https://github.com/arrow-py/arrow) - Better dates & times for Python.
+- [Bokeh](https://github.com/bokeh/bokeh) - Interactive web plotting for Python.
+- [Crossbar](https://github.com/crossbario/crossbar/) - Open-source Unified Application Router (Websocket & WAMP for Python on Autobahn).
+- [Datasette](https://github.com/simonw/datasette) - An open source multi-tool for exploring and publishing data.
+- [dejavu](https://github.com/worldveil/dejavu) - Audio fingerprinting and recognition.
+- [djedi-cms](http://djedi-cms.org/) - A lightweight but yet powerful Django CMS with plugins, inline editing and performance in mind.
+- [Dramatiq](https://github.com/Bogdanp/dramatiq) - A fast and reliable background task processing library for Python 3.
+- [FastAPI Users](https://github.com/fastapi-users/fastapi-users) - Ready-to-use and customizable users management for FastAPI.
+- [Faker](https://github.com/joke2k/faker) - Faker is a Python package that generates fake data for you.
+- [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) - Fuzzy String Matching.
```
Comment (stocks29) [inline docs/python.md]: this project moved to: https://github.com/seatgeek/thefuzz

could you please update to the latest?

File: docs/python.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+- [HTTPie](https://github.com/httpie/httpie) - Modern command line HTTP client, user-friendly curl alternative.
+- [mrq](https://github.com/pricingassistant/mrq) - Mr. Queue - A distributed worker task queue in Python using Redis & gevent.
+- [p](https://github.com/qw3rtman/p) - Dead simple interactive Python version management.
+- [PeachPy](https://github.com/Maratyszcza/PeachPy) - x86-64 assembler embedded in Python. Can be used as inline assembler for Python or as a stand-alone assembler for Windows, Linux, OS X, Native Client and Go.
+- [Pendulum](https://github.com/sdispater/pendulum) - Python datetimes made easy.
+- [Peewee](https://github.com/coleifer/peewee) - A small, expressive ORM.
+- [Poetry](https://python-poetry.org/) - Python dependency management and packaging made easy.
+- [Pony ORM](https://github.com/ponyorm/pony) - ORM that provides a generator-oriented interface to write database queries.
+- [Pydantic](https://github.com/pydantic/pydantic) - Data validation and settings management using Python type annotations.
+- [pyocr](https://github.com/jflesch/pyocr) - A wrapper for Tesseract and Cuneiform.
```
Comment (stocks29) [inline docs/python.md]: this project has moved to: https://gitlab.gnome.org/World/OpenPaperwork/pyocr

could you please update this?

File: docs/python.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+- [pyquery](https://github.com/gawel/pyquery) - A jQuery-like library for parsing HTML.
+- [Pyro](https://github.com/pyro-ppl/pyro) - Deep universal probabilistic programming with Python and PyTorch.
+- [Rich](https://github.com/Textualize/rich) - Rich is a Python library for rich text and beautiful formatting in the terminal.
+- [Streamlit](https://github.com/streamlit/streamlit) - Streamlit lets you create apps for your machine learning projects with simple Python scripts.
+- [TinyDB](https://github.com/msiemens/tinydb) - TinyDB is a lightweight document-oriented database optimized for your happiness.
+- [Typer](https://github.com/tiangolo/typer) - Typer, build great CLIs. Easy to code. Based on Python type hints.
+- [watchdog](https://github.com/gorakhargosh/watchdog) - API and shell utilities to monitor file system events.
+- [Whoosh](http://whoosh.readthedocs.io/) - A fast, pure Python search engine library.
+- [Zappa](https://github.com/Miserlou/Zappa) - Serverless Python.
```
Comment (stocks29) [inline docs/python.md]: this project has moved to: https://github.com/zappa/Zappa

could you please update this?

**Review by stocks29 — CHANGES_REQUESTED** [review]
@SherbetLemon47 thanks for submitting this PR! I added a few inline comments. Also since python is heavy into machine learning, could you please add a few ML libs?

**Comment by SherbetLemon47** [comment]
Updated all of the above. Will add a few ML libs

File: docs/python.md (hunk 2 above)
Comment (SherbetLemon47) [reply docs/python.md]: Its not as frequently updated, so removed in favor of pytesseract.

**Comment by SherbetLemon47** [comment]
Added some popular and useful ones!

File: docs/python.md [hunk 4]
```diff
@@ -1,33 +1,45 @@
...
 - [Datasette](https://github.com/simonw/datasette) - An open source multi-tool for exploring and publishing data.
 - [dejavu](https://github.com/worldveil/dejavu) - Audio fingerprinting and recognition.
 - [djedi-cms](http://djedi-cms.org/) - A lightweight but yet powerful Django CMS with plugins, inline editing and performance in mind.
 - [Dramatiq](https://github.com/Bogdanp/dramatiq) - A fast and reliable background task processing library for Python 3.
 - [FastAPI Users](https://github.com/fastapi-users/fastapi-users) - Ready-to-use and customizable users management for FastAPI.
 - [Faker](https://github.com/joke2k/faker) - Faker is a Python package that generates fake data for you.
-- [fuzzywuzzy](https://github.com/seatgeek/fuzzywuzzy) - Fuzzy String Matching.
+- [Fastai](https://github.com/fastai/fastai) - A deep learning library that simplifies training fast and accurate neural networks using modern best practices, built on top of PyTorch.
+- [fuzzywuzzy](https://github.com/seatgeek/tchefuzz) - Fuzzy String Matching.
```
Comment (stocks29) [inline docs/python.md]: minor typo

```suggestion
- [fuzzywuzzy](https://github.com/seatgeek/thefuzz) - Fuzzy String Matching.
```

**Review by stocks29 — COMMENTED** [review]
@SherbetLemon47 thanks for updating this. just one minor typo that needs fixing and then this is good to go.

**Review by stocks29 — APPROVED** [review]

**Pull request merged**